tests/e2e/tools/lib/post.js

```diff
@@ -48,23 +48,32 @@ async function publishPostAndVisit({page, editor}) {
  * Creates a new post with a featured image set.
  *
  * @param {Object} p - Parameters for creating the post and setting the featured image.
+ * @param {Object} p.page - The page object used to interact with the editor.
```

Comment:
Not necessarily for this PR, but we should harmonise our comments, in other places we use `Page`, `Editor`, etc